opencog/reasoning/RuleEngine/rule-engine-src/README.md

```diff
@@ -0,0 +1,169 @@
+Unified rule engine
+-------------------
+
+**Introduction**
+
+   The unified rule engine project aims at building a generic opencog rule engine on top of the pattern matcher with a c++ PLN
```

Comment:
another minor comment: wrapping long lines to be 72 chars wide make this more readable in raw-text form.